app/assets/stylesheets/moderators.scss

```diff
@@ -2,163 +2,104 @@
-    }
-    @media screen and (min-width: 960px) {
-      width: calc(190px + 11vw);
-      padding: 80px 0px;
-      margin: 15px 2px;
-      .mod-index-hero-data-pod-number {
-        font-size: calc(2.7em + 0.2vw);
-      }
-    }
-    @media screen and (min-width: 1300px) {
-      width: calc(260px + 8vw);
-      padding: 80px 0px;
-      margin: 30px;
-      .mod-index-hero-data-pod-number {
-        font-size: calc(2.8em + 0.5vw);
-      }
-    }
-  }
-  .mod-index-hero-sub {
-    font-size: 0.9em;
-  }
+  padding: 30px 10px;
 }
 
-.mod-index-header {
-  margin: 50px auto 30px;
-  width: 96%;
-  max-width: 1600px;
-  padding-left: 20px;
-  .mod-index-header-top-filters {
-    padding: 10px 0px 25px;
-    a {
-      color: var(--body-color);
-      font-size: 0.86em;
-      padding: 3px 18px;
-      border-radius: 100px;
-      border: 1px solid transparent;
-      &.mod-index-top-filter-link--active {
-        border: 1px solid var(--body-color);
-      }
-    }
-  }
-  a.mod-index-header-link {
-    color: $black;
-    padding: 5px 15px;
-    border-radius: 3px;
-    font-size: 1.5em;
-    font-weight: bold;
-    border: 1px solid transparent;
-  }
-  a.mod-index-header-link--active {
-    @include themeable(border, theme-border, 1px solid $black);
-  }
+.mod-center-heading {
+  margin-bottom: var(--su-6);
 }
 
-.mod-index-list {
-  margin: 5px auto;
-  width: 96%;
-  max-width: 1600px;
-  border-radius: 3px;
-  border: 1px solid var(--card-color-tertiary);
-  background: var(--card-bg);
+.mod-index-container {
+  .mod-index-header-new {
+    background-color: var(--base-inverted);
+    border-radius: var(--radius) var(--radius) 0 0;
+    padding: var(--su-6) var(--su-4);
 
-  .mod-index-row {
-    padding: 10px;
-    @include themeable(
-      border-bottom,
-      theme-border,
-      1px solid $light-medium-gray
-    );
-    a.mod-index-article-title {
-      font-size: calc(1.2em + 0.3vw);
-      font-weight: bold;
-      margin-right: 6px;
-      display: inline-block;
-      color: var(--body-color);
-    }
-    .mod-index-article-tagslist {
-      display: inline-block;
-    }
-    a.mod-index-article-tag {
-      font-size: 0.9em;
-      padding: 3px 8px;
-      border-radius: 3px;
-      background: $light-gray;
-      color: $black;
-      vertical-align: 0.2em;
+    .tag-view {
+      font-size: var(--fs-3xl);
+      font-weight: var(--fw-bold);
+      color: var(--base-90);
+      line-height: var(--lh-base);
     }
-    .mod-index-actions-and-details {
-      img {
-        height: 1.4em;
-        width: 1.4em;
-        border-radius: 100px;
-        margin-right: 5px;
-        display: inline-block;
-        vertical-align: -0.3em;
-      }
-      a {
-        font-weight: 500;
-        color: var(--card-color-tertiary);
+  }
+  main.mod-index-list {
+    border: 1px solid var(--base-10);
+    margin: 0;
+    width: 100%;
+    .mod-index-list-header {
+      display: grid;
+      grid-template-columns: 4fr 2fr 1fr;
```

Comment:
Is this good for smaller mobile as well, or is the mod center currently only being considered for Desktop, laptops, tablets?

Reply:
Pretty much only for desktop. Based on user research they strictly use it for desktop.

For this cycle we decided that mobile view will be a nice-to-have.